tests/lib/cli_util_test.py

```diff
@@ -59,6 +59,10 @@ def test_parse_key_target(self):
         for spec, expected_parse in cases:
             self.assertEqual(cli_util.parse_key_target(spec), expected_parse)
 
+        usage_error_cases = [':', 'a:']
```

Comment:
Should we add `''` in the array too?

Reply:
See this link: https://github.com/codalab/codalab-worksheets/pull/2049#discussion_r391375266

Reply:
I don't think that comment changes the fact that we still want to throw an error when an empty string is passed to `parse_key_target`?

Reply:
Never mind, I was thinking of something else. We should add an empty string to this test suit.